### ISSUE TITLE
Custom cipher suites order is not respected

### ISSUE THREAD

**Issue by JavaScriptGangstaCat** [OtterBrowser/otter-browser#972]
In the Advanced settings' Security panel, one can choose the cipher suites he wants, and in the order he wants. But, the order is not respected by the browser when surfing, and Otter orders the cipher suites by "strength": ECDH key exchange 256 Bit, then RSA key exchange 256 Bit, then ECDH key exchange 128 Bit, then RSA key exchange 128 Bit, etc. When the user choose a particular order, then apply the settings, Otter orders the cipher suites by "strength" and forget the user's ordering (except in the otter.conf file where the order is respected). You can see it by re-opening the Settings window to see the new ordering.

This behavior can be observed on a website like https://cc.dcsec.uni-hannover.de/ which order the cipher suites by the preferred order sent by the browser.

Otter should respect the preference order the user choose, for example when someone want AES-128-GCM in priority than AES-256-CBC.

Otter 0.9.08 weekly `#95` Windows x64.

**Comment by Emdek** [comment]
@JavaScriptGangstaCat, could you please check if that solves this issue for you?

**Comment by JavaScriptGangstaCat** [comment]
@Emdek, the bug is still here in #98. Try to put a 128 bit cipher at the top of the list in the Settings then Apply, you will see that Otter re-organize the list by strength directly in the application. However, the otter.conf file has the good order.

**Comment by Emdek** [comment]
@JavaScriptGangstaCat, that change will be available only in weekly ``#99``, order should be passed correctly since that commit, so if it won't get respected by QtNetwork then this ticket will be marked as upstream issue.
Note that since yesterday you can check which cipher was used using GUI (#1000). ;-)

**Comment by JavaScriptGangstaCat** [comment]
@Emdek, in `#99`, when I modify the ciphers list, even when I don't change the order but just delete some insecure ciphers like RC4 and MD5 ones, I get the error:

```Error 99
Invalid or empty cipher list (error:1410D0B9:SSL routines:SSL_CTX_set_cipher_list:no cipher match)```
